docs/assets/js/charts.js

```diff
@@ -115,10 +115,18 @@ var stackedBarChartDefaults =
 	maintainAspectRatio: false
 };
 
+function createSpinner(canvas)
+{
+	var parent = $("<div style=\"position:absolute;height:100%;width:100%;\"></div>");
```

Comment:
Here and in the other places below I think we should rather use `let` to declare the variables, no? I know this is not yet consistent throughout the codebase (... and that's why I love you [linting proposal](#58)).

Does this sound good to you? If yes, then you don't need to to anything. I would just adjust your commit on merge.

Reply:
Sounds fine to me! Whatever style the project decides on, I will conform to. 👍 

Once I start putting together the JS test running and linting changes I have planned, we can discuss in finer detail the particular JS style rules the project wants to adopt.